example/lib/barcode_scanner_window.dart

```diff
@@ -19,46 +19,114 @@ class _BarcodeScannerWithScanWindowState
-          ),
-          Align(
-            alignment: Alignment.bottomCenter,
-            child: Container(
-              alignment: Alignment.center,
-              padding: const EdgeInsets.symmetric(horizontal: 16, vertical: 8),
-              height: 100,
-              color: const Color.fromRGBO(0, 0, 0, 0.4),
-              child: ScannedBarcodeLabel(barcodes: controller.barcodes),
+      body: Container(
+        height: screenSize.height,
+        width: screenSize.width,
+        color: Colors.blueGrey,
+        child: Stack(
+          children: [
+            const Text(
+              "Background",
+              style: TextStyle(
+                color: Colors.white,
+                fontSize: 30,
+                fontWeight: FontWeight.bold,
+              ),
             ),
-          ),
-        ],
+            Center(
+              child: Container(
+                width: containerWidth,
+                height: containerHeight,
+                color: Colors.red,
+                child: Stack(
+                  fit: StackFit.expand,
+                  children: [
+                    MobileScanner(
+                      fit: boxFit,
+                      scanWindow: scanWindow,
+                      controller: controller,
+                      errorBuilder: (context, error) {
+                        return ScannerErrorWidget(error: error);
+                      },
+                    ),
+                    BarcodeOverlay(controller: controller, boxFit: boxFit),
+                    ScanWindowOverlay(
+                      scanWindow: scanWindow,
+                      controller: controller,
+                    ),
+                    Align(
+                      alignment: Alignment.bottomCenter,
+                      child: Container(
+                        alignment: Alignment.center,
+                        padding: const EdgeInsets.symmetric(
+                          horizontal: 16,
+                          vertical: 8,
+                        ),
+                        height: 100,
+                        color: const Color.fromRGBO(0, 0, 0, 0.4),
+                        child:
+                            ScannedBarcodeLabel(barcodes: controller.barcodes),
+                      ),
+                    ),
+                    Text(
+                      "Camera preview\nBoxFit: $boxFit",
+                      style: const TextStyle(
+                        color: Colors.white,
+                        fontSize: 20,
+                        fontWeight: FontWeight.bold,
+                      ),
+                    ),
+                    Positioned(
+                      bottom: 100,
+                      width: containerWidth,
+                      child: Wrap(
+                        spacing: 0.5,
+                        runSpacing: 3.0,
+                        alignment: WrapAlignment.center,
+                        children: BoxFit.values.map((fit) {
```

Comment:
You can use a tear off here:

```suggestion
                        children: BoxFit.values.map(_buildBoxFitButton).toList(),
```